File: fibonacci.cs
namespace Fibonacci;

class Program {
  static int Fibonacci(int n) {
    if (n <= 1) return n;
    else return Fibonacci(n - 1) + Fibonacci(n - 2);
  }
  
  static void Main() {
    Fibonacci(5); //5
  }
}

File: Fibonacci.java
// Quick note, I'm learning Java and I just guessed that this would be the same as C#.
// It's crazy that I was right. This code was a guess and I got it right lol

public class Fibonacci {
    public static int fibonacci(int n) {
        if (n <= 1) return n;
        else return fibonacci(n - 1) + fibonacci(n - 2);
    }
    public static void main(String[] args) {
        fibonacci(5); //5
    }
}
